### ISSUE TITLE
static ports on devserver for multiple modules

### ISSUE THREAD

**Issue by crazystick** [GoogleCloudPlatform/app-maven-plugin#436]
I think I already know the answer based on #330 and #435 but I want to know if anyone has a workaround for this. I've just migrated to the new tooling (was painful, documentation is not good) and I have this problem.

We use an nginx reverse proxy configuration to replace what the dispatch configuration does in production. It used to work great but now all the modules except the default run on random ports.

In addition, setting devserver.host to 0.0.0.0 doesn't seem to make the server listen on all the interfaces.

Hoping someone has a workaround for this, as it feels like a big step backwards.

**Comment by loosebazooka** [comment]
@saturnism @ludoch any ideas on the devserver config?

**Comment by chanseokoh** [comment]
Can you try and see if this workaround works? https://github.com/GoogleCloudPlatform/app-maven-plugin/issues/235#issuecomment-356014144

**Comment by crazystick** [comment]
@chanseokoh yes that used to work with the old devserver, but doesn't seem to work with the new version based on the Cloud SDK

**Comment by loosebazooka** [comment]
The current version of this plugin uses the *old* dev server
